src/converter-gen/mvn-converters-xpl.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
Blocking, but minor: Wendell, can you run the following command from WSL or Git Bash to ensure the file is marked executable and not break the execution wrapper script (or everyone will have to do this every checkout and in CI/CD):

```sh
chmod +x path/to/repo/metaschema-xslt/src/converter-gen/mvn-converters-xpl.sh
git add !$
git commit -m "Mark mvn-converters-xpl.sh executable for general use."
```

Reply:
```bash
wap1@PG902643 MINGW64 ~/Documents/usnistgov/metaschema-xslt (issue43-converter-gen-scripting *%>)
$ chmod +x src/converter-gen/mvn-converters-xpl.sh

wap1@PG902643 MINGW64 ~/Documents/usnistgov/metaschema-xslt (issue43-converter-gen-scripting %=)
$ git add !$
git add src/converter-gen/mvn-converters-xpl.sh

wap1@PG902643 MINGW64 ~/Documents/usnistgov/metaschema-xslt (issue43-converter-gen-scripting %=)
$ git commit -m "Mark mvn-converters-xpl.sh executable for general use."
On branch issue43-converter-gen-scripting
Your branch is up to date with 'wendellpiez/issue43-converter-gen-scripting'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        docs/
        src/METASCHEMA-TOOLS-wip.xpl
        src/testing/docs/

nothing added to commit but untracked files present (use "git add" to track)

wap1@PG902643 MINGW64 ~/Documents/usnistgov/metaschema-xslt (issue43-converter-gen-scripting %=)
$

```

Reply:
Hmmm that is odd. I guess I can try assisting later if you'd like, but it seems that is done from Cygwin (Git Bash) not WSL and maybe it ignores ACL directives like the executable bit? If you try in WSL and not Git Bash does the same thing happen?

(I only ask about this because I forgot it once in one of our projects and it got merged, for a while I had to remind people do this themselves every time they checked out and I didn't make friends with this, sorry.)

Reply:
I think that was in WSL, but not certain as of this moment.

No biggie, we will persist and get it right! LMK.

Reply:
I tried it and worked, so ... 🤷 ?